src/components/Popover/Arrow.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  Arrow as RadixPopoverArrow,
+  PopoverArrowProps as RadixPopoverArrowProps,
+} from "@radix-ui/react-popover";
+import { vars } from "~/theme";
+import { classNames } from "~/utils";
+import { arrow } from "./Popover.css";
+
+export type PopoverArrowProps = RadixPopoverArrowProps & {
+  backgroundColor?: string;
+  borderColor?: string;
+};
+
+export const Arrow = ({
+  className,
+  backgroundColor = vars.colors.background.subdued,
+  borderColor = vars.colors.border.neutralPlain,
+  ...props
+}: PopoverArrowProps) => {
+  return (
+    <RadixPopoverArrow
+      {...props}
+      className={classNames(arrow, className)}
+      asChild
+    >
+      <svg
+        width="20"
+        height="9"
+        viewBox="0 0 20 9"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M8.08579 7.08579L0.5 -0.5H18.5L10.9142 7.08579C10.1332 7.86683 8.86684 7.86684 8.08579 7.08579Z"
+          strokeLinejoin="round"
+          fill={backgroundColor}
+          stroke={borderColor}
+        />
+      </svg>
```

Comment:
**suggestion**:  You can draw it with using  pure css, there is no need to use dedicated svg

Reply:
<img width="759" alt="image" src="https://user-images.githubusercontent.com/41952692/228779623-9d46413a-d402-49c7-b358-e04948a9156b.png">
Achieving this shape in pure CSS is a complex task - main idea is to create rounded triangle using borders on 0x0 element, and clip it with inner triangle of different color, which creates an illusion of triangle with a border, and then rotate everything by 45°. Positioning of the inner triangle may be somewhat difficult, I think it's gonna be prone to visual bugs when scale is changed, and debugging it would be more time-consuming than just replacing the svg here.